.gitallowed

```diff
@@ -0,0 +1,2 @@
+AKIAIOSFODNN7EXAMPLE
```

Comment:
Can you remove this file from the change please? If you want to propose this be added, I'd prefer it in a separate pull request.